addons/mail/static/tests/thread/attachment_list_tests.js

```diff
@@ -171,6 +171,38 @@ QUnit.test("view attachment", async () => {
     await contains(".o-FileViewer");
 });
 
+QUnit.test("view link-type pdf attachment", async (assert) => {
```

Comment:
Would rename to "can view pdf url"

Short and very easy to grasp the functional purpose with easy wording.